<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QPushButton" name="button_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_0">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_plus">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_star">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_equal">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>580</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="output_lcd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="input_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name='text'>
     <string>Input: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="readOnly">
     <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clear">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>580</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="pre_fix_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>205</y>
     <width>100</width>
     <height>15</height>
    </rect>
   </property>
   <property name='text'>
     <string>Pre Fix Notation: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pre_fix_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Pre Fix Notation</string>
   </property>
  </widget>
  <widget class="QLabel" name="post_fix_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>255</y>
     <width>110</width>
     <height>15</height>
    </rect>
   </property>
   <property name='text'>
     <string>Post Fix Notation: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="post_fix_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Post Fix Notation</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
